hub/scripts.js

```diff
@@ -415,8 +415,17 @@
    * Page details are updated based on these values.
    */
   function handlePageDetails() {
-    const paths = window.location.pathname.replace('/bench', '')
+    const path = window.location.pathname;
+    let paths;
+
+    if (path.indexOf('_draft')) {
+      paths = path.replace('/bench' && '_draft', '')
```

Comment:
I don't think this will work like you intend to

Reply:
Hi @narcis-radu  I've changed the replace
This is the test url https://localnav-ondrafts--fedpub--elaineskpt.hlx.page/_draft/de/documentcloud/acrobat/hub/how-to/document.html 
Getting the de nav on drafts too now

Reply:
Thank you @elaineskpt, it works now. I'd like one more change since the if clause on L#421 is not needed anymore. You can keep the `const path` and chain the `replace` methods as needed without checking whether the path contains `_draft` or not.

Reply:
Sure I changed that. I though we had to check and replace only if we have _draft